client/.eslintrc.json

```diff
@@ -1,11 +1,14 @@
 {
-  "extends":["next/core-web-vitals",
+  "extends": [
+    "next/core-web-vitals",
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
-    "plugin:@typescript-eslint/recommended"],
+    "plugin:@typescript-eslint/recommended"
+  ],
   "rules": {
     "@typescript-eslint/no-unused-vars": "error",
     "react/display-name": "off",
-    "react-hooks/exhaustive-deps": "off"
+    "react-hooks/exhaustive-deps": "off",
+    "@next/next/no-img-element": "off"
```

Comment:
IMO this rule shouldn't be enforced since we should be allowed to use vanilla <img> tag.

Reply:
Feel free to adapt the rules to what you think is best

Reply:
@dancespiele you set up the linter, are you ok with that?